### ISSUE TITLE
git-lfs integration broken with git-lfs 2.1.0

### ISSUE THREAD

**Issue by asottile** [pre-commit/pre-commit-hooks#195]
More of an fyi than anything else, upstream broke `git lfs status --porcelain` in _some_ cases.  Making this ticket first and then I'll link the upstream one :)

you can see test failures here:

https://ci.appveyor.com/project/asottile/pre-commit-hooks/build/job/u3i1orog0o6buwnl

**Comment by asottile** [comment]
It also appears git lfs status is now including **non lfs files**.  I've opened a new PR to fix `--json` at least: https://github.com/git-lfs/git-lfs/pull/2374

**Comment by asottile** [comment]
Here's a WIP branch: https://github.com/pre-commit/pre-commit-hooks/compare/use_new_git_lfs?expand=1

**Comment by asottile** [comment]
Now waiting on https://github.com/appveyor/ci/issues/1655

**Comment by asottile** [comment]
This has been fixed from the appveyor side!  Now to fix the problems that have come up while appveyor was broken :)
